frontend/src/components/Articles.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { useTable, useSortBy, usePagination } from "react-table";
+
+const Table = ({ columns, data }) => {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    prepareRow,
+
+    page, // Instead of using 'rows', we'll use page,
+    // which has only the rows for the active page
+
+    // The rest of these things are super handy, too ;)
```

Comment:
Do we need these comments? Would it be better to remove them and the extra spaces in between?

```suggestion
    page,
```

Reply:
I removed this file entirely since were not using it at all aha. just completely forgot that it was there when i made the pr.